catalogs/.dockerignore

```diff
@@ -0,0 +1,3 @@
+.dockerignore
+Dockerfile
+README.md
```

Comment:
`.dockerignore` prevents files from being shipped to the build context, which is useful if the build context would be excessively large, but I don't see a huge benefit to ignoring a few text files when this isn't standard practice.

Reply:
Yeah, agreed that the benefit is marginal. Just a general habit of mine to strip out anything that isn't necessary for the final image. Should this be removed for sake of consistency with other projects?

Reply:
> Should this be removed for sake of consistency with other projects?
That's what I'm thinking. It feels like a red herring, in the sense that other places we've used it there was a unique constraint we were trying to get past. Either that, or we start pressing to include this as standard in other places where we build images.

Reply:
> Either that, or we start pressing to include this as standard in other places where we build images.

This is tempting, but I don't feel like this project is the right place to kick that off. I think targeting application templates could be more promising though! I'll get this taken out of this PR.